theme/lumo/vaadin-grid-pro-edit-text-field.html

```diff
@@ -3,9 +3,9 @@
 <link rel="import" href="../../../vaadin-lumo-styles/typography.html">
 <link rel="import" href="../../../vaadin-text-field/theme/lumo/vaadin-text-field.html">
 
-<dom-module id="lumo-grid-pro-edit-text-field" theme-for="vaadin-grid-pro-edit-text-field">
```

Comment:
Let's rename this to something shorter and more common, like `lumo-grid-pro-editor`. It might be that we will reuse that later for other components, like `vaadin-password-field` etc.

Reply:
> Can you check whether this should be also fixed for Material?

I checked that at least this cursor issue doesn't affect Material since `material-text-field` styles don't set a cursor style.

We could do the same change (so that `material-text-field` styles aren't added into `vaadin-select-text-field` after `material-select-text-field` styles) for consistency but I'm not sure if it would change any of the rendered styles in this case (when using Material theme).

> Let's rename this to something shorter and more common, like `lumo-grid-pro-editor`. It might be that we will reuse that later for other components, like `vaadin-password-field` etc.

I didn't want to rename this module (`lumo-grid-pro-edit-text-field`) since I wasn't sure if that is used in any other components. It was named like that before my changes. I just moved most of its content into a new module named `lumo-grid-pro-edit-text-field-no-deps` (which is now also included into the original). That new one could at least be renamed if you think it makes sense.

Reply:
Yes, what I mean is renaming the new module as `-no-deps` is not much self-descriptive.

Reply:
Ok, I renamed the new style module as you suggested.